### PR TITLE
Fixed: Remove non-functional filters for Trakt Lists

### PR DESCRIPTION
**Pull request by stevietv**
Cherry pick of https://github.com/Radarr/Radarr/commit/5d8775ac96d4b7de1b4381c463e71cf035a3f942

#### Database Migration
NO

#### Description
backport from Radarr reference https://github.com/Radarr/Radarr/issues/7464

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs [hunk 1]
```diff
@@ -23,5 +43,17 @@ public TraktPopularSettings()
 
         [FieldDefinition(1, Label = "List Type", Type = FieldType.Select, SelectOptions = typeof(TraktPopularListType), HelpText = "Type of list you're seeking to import from")]
         public int TraktListType { get; set; }
+
+        [FieldDefinition(2, Label = "Rating", HelpText = "Filter movies by rating range (0-100)")]
```
Comment (Qstick) [inline src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs:47]: not `movies`

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs [hunk 2]
```diff
@@ -23,5 +43,17 @@ public TraktPopularSettings()
...
+        public string Rating { get; set; }
+
+        [FieldDefinition(3, Label = "Certification", HelpText = "Filter movies by a certification (NR,G,PG,PG-13,R,NC-17), (Comma Separated)")]
```
Comment (Qstick) [inline src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs:50]: we don't want `movies` or movie certs

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs [hunk 3]
```diff
@@ -23,5 +43,17 @@ public TraktPopularSettings()
...
+        public string Certification { get; set; }
+
+        [FieldDefinition(4, Label = "Genres", HelpText = "Filter movies by Trakt Genre Slug (Comma Separated) Only for Popular Lists")]
```
Comment (Qstick) [inline src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs:53]: not `movies`

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs [hunk 4]
```diff
@@ -23,5 +43,17 @@ public TraktPopularSettings()
...
+        public string Genres { get; set; }
+
+        [FieldDefinition(5, Label = "Years", HelpText = "Filter movies by year or year range")]
```
Comment (Qstick) [inline src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs:56]: not `movies`

**Review by Qstick — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs (hunk 1 above)
Comment (stevietv) [reply src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs]: edited

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs (hunk 2 above)
Comment (stevietv) [reply src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs]: removed certs filter

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs (hunk 3 above)
Comment (stevietv) [reply src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs]: edited

File: src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs (hunk 4 above)
Comment (stevietv) [reply src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularSettings.cs]: edited

**Pull request merged**